gp-easy-passthrough/gpep-edit-entry.php

```diff
@@ -35,6 +35,8 @@ public function __construct( $options ) {
 		// Bypass limit submissions on validation
 		add_filter( 'gform_validation', array( $this, 'bypass_limit_submission_validation' ) );
 
+		// Update entry values on submission
+		add_action( 'gform_after_submission', array( $this, 'gpep_update_entry' ), 10, 2 );
```

Comment:
Do you know the reason technically why this is needed? It should already be updating the entry thanks to https://github.com/gravitywiz/snippet-library/pull/578/files#diff-e8542d5676691b24606b326cc6bc02c9ac4c7990632c02bb5133f709918fd0f7R30

If this is needed, we'll want to make sure it's scoped to this particular form.

Reply:
The link just points me back to the git compare of this branch. From what I recall, the entry data wasn't matching post edit, which is where the idea of "edit entry" firing GFAPI update came.